python/pyspark/sql/udf.py

```diff
@@ -56,6 +56,10 @@ def _create_udf(f, returnType, evalType):
     return udf_obj._wrapped()
 
 
+class UDFColumn(Column):
```

Comment:
BTW, what do you think about adding an attribute instead?

Reply:
Removed